gulpfile.js

```diff
@@ -150,5 +160,5 @@ function log(error) {
 }
 
 
-gulp.task('default', ['dist', 'copy']);
+gulp.task('default', ['dist', 'copy', 'deploy']);
```

Comment:
@chrisjlebron Huh, interesting.  The impression I get is it's the opposite: https://github.com/gulpjs/gulp/blob/master/docs/API.md#async-task-support

Anyway, seems to work as advertised.  Are you ok with this?